scripts/client.js

```diff
@@ -0,0 +1,25 @@
+import KintoClient from "kinto-client";
+
+
+let client;
```

Comment:
I wonder if we shouldn't use a class that we can instanciate rather than a sigleton like that.